test/test_artik_craft.js

```
@@ -67,5 +68,6 @@ describe('ARTIK - craft ai integration', () => {
       chai.expect(res).to.equal('green');
       done();
     })
+    .catch(console.log)
```

Arg, this will silently ignore failures